EN: Jet Boots item description translation. (#89)
Arks-Layer/PSO2esTranslations: json/Item_Weapon_JetBoots.txt

		"jp_text": "ジェットブーツ",
		"tr_text": "Jet Boots",
		"jp_explain": "アークスで正式採用されている魔装脚。\n高速でテクニックを行使できる\n初心者に扱いやすい入門用モデル。",

> 		"tr_explain": "Jet Boots used by ARKS. \nAllow use of high-speed techniques. \nAn inexpensive model intended for newcomers."

Reviewer: You couldn't have known cause I forgot to say, but when a weapon like this is part of a multi-weapon-type series, we want to keep the wording as much the same as possible across each. In this case, it would start:
`Jet Boots adopted by ARKS.`
It can be a little annoying to find examples of weapons in the same series, but a good go-to is swords (first weapon type in any given list so it's where people tend to start) or launchers (my pet weapon).

Also, your last line here is too long to be made to fit into the description box, so it'll need a little rewriting. Don't be afraid to merge two sentences together if you have to to save space.

Some weapons, including this one, have OT and NT versions. For these weapons, the NT description is the same as the OT, so feel free to do both at once.

As a couple of minor nitpicks, we prefer to capitalise Techniques, and to eliminate spaces before line breaks.

```suggestion
		"tr_explain": "Jet Boots adopted by ARKS. An\ninexpensive model that lets beginners\nuse high-speed Techniques."
```

	},
	{
		"assign": "11601",
		"jp_text": "アルバＪブーツ",
		"tr_text": "Alva Jet Boots",
		"jp_explain": "アークスで使用される改良型魔装脚。\n扱い易いサイズを保ったまま\n出力を向上させた強化モデル。",

> 		"tr_explain": "Improved Jet Boots used by ARKS.\nImproved output while \nkeeping an easy-to-use size."

Reviewer: You actually have some room to breathe on this one, it doesn't need to be as succinct as you've made it.
```suggestion
		"tr_explain": "Improved Jet Boots used by ARKS.\nOutput has been enhanced while\nkeeping to an easy-to-use size."
```

	},
	{
		"assign": "11602",
		"jp_text": "ヴィタＪブーツ",
		"tr_text": "Vita Jet Boots",
		"jp_explain": "基礎モデルに改良を加えた魔装脚。\nかかとのブレードをサイズアップし\n攻撃力とフォトン干渉力が向上した。",

> 		"tr_explain": "A Jet Boots model improved from basic design.\nIncreased heel blade size and \nimproved Photon output."

Reviewer: See above; this would start `Jet Boots that further improve on\nthe base model.`.
```suggestion
		"tr_explain": "Jet Boots that further improve on\nthe base model. The increased heel\nblade size boosts Photon output."
```

	},
	{
		"assign": "11609",
		"jp_text": "ジェットスライダー",
		"tr_text": "Jet Slider",
		"jp_explain": "滑らかなブレードが特徴的な魔装脚。\n二枚のブレードの役割を明確に\n分けることで、高い安定性を誇る。",

> 		"tr_explain": "Jet Boots with characteristic smooth blades. \nThe blades precisely divide into two parts \nto boost stability."

Reviewer: Your first and second lines are too long, but rearranging the line breaks will take care of that, the actual text will fit just fine.
```suggestion
		"tr_explain": "Jet Boots with characteristic smooth\nblades. The blades divide neatly into\ntwo parts to boost stability."
```

	},
	{
		"assign": "116010",
		"jp_text": "アルバＪスライダー",
		"tr_text": "Alva Jet Slider",
		"jp_explain": "滑らかなブレードを持つ魔装脚改良型。\nブレードの接続部を大型化することで\n攻撃範囲を広げている。",

> 		"tr_explain": "Improved Jet Boots with smooth blades.\nThe blades' surface was increased to \nboost attack range."

Reviewer: Line breaks need rearranging
```suggestion
		"tr_explain": "Improved Jet Boots with smooth\nblades. The blades' surface area was\nincreased to boost attack range."
```

	},
	{
		"assign": "116011",
		"jp_text": "ヴィタＪスライダー",
		"tr_text": "Vita Jet Slider",
		"jp_explain": "滑らかなブレードを持つ魔装脚最終形。\n高出力だが鋭いブレードの扱いが難しく\n熟練のアークスに好まれている。",

> 		"tr_explain": "Latest model of Jet Boots with smooth blades. \nIncreased output given by sharpened blades \nis favored by more skilled ARKS."

Reviewer: This is gonna need shortening.
```suggestion
		"tr_explain": "Latest model of Jet Boots with\nsmooth blades. The sharpened blades\nare favored by more skilled ARKS."
```

	},
	{
		"assign": "116012",
		"jp_text": "ジェットビースオート",
		"tr_text": "Jet Beasauto",
		"jp_explain": "鋭利な三枚のブレードを持つ魔装脚。\n非常に軽量だがフォルムが平坦で\n扱いに慣れるまでが難しい。",

> 		"tr_explain": "Jet Boots with three sharp blades. \nYou have to get used to their \nunusual smoothness and lightness."

Reviewer: Spaces before line breaks aside, this is perfect!
```suggestion
		"tr_explain": "Jet Boots with three sharp blades.\nYou have to get used to their\nunusual smoothness and lightness."
```

	},
	{
		"assign": "116013",
		"jp_text": "アルバＪビースオート",
		"tr_text": "Alva Jet Beasauto",
		"jp_explain": "複数ブレードを持つ魔装脚の改良型。\n三枚のブレードが強化され\n側面にも一枚ブレードが追加された。",

> 		"tr_explain": "Improved Jet Boots with multiple blades. \nTheir blades were strenghened \nand additional blade was added."

Reviewer: This is just a bit too long, also spelling on 'strengthened'. Sometimes the best thing to do is to rearrange the concepts within a sentence just to get the words to fit.
```suggestion
		"tr_explain": "Improved Jet Boots with multiple\nblades. An additional blade has been\nadded, and all have been enhanced."
```

	},
	{
		"assign": "116014",
		"jp_text": "ヴィタＪビースオート",
		"tr_text": "Vita Jet Beasauto",
		"jp_explain": "複数ブレードを持つ魔装脚の最終形。\n全ブレードを大型化し、軽量なままに\n大幅な出力の向上に成功した。",

> 		"tr_explain": "Latest Jest Boots with multiple blades. \nAll blades have increased size while maintaining \nlightness successfully boosting their output."

Reviewer: This is quite a bit too long, also spelling on 'Jet Boots'.
```suggestion
		"tr_explain": "Latest Jet Boots with multiple blades.\nThe blades have been increased in\nsize without increasing their weight."
```

		"jp_text": "ラムダレザスイフト",
		"tr_text": "Lambda Lezaswift",
		"jp_explain": "新素材で開発された魔装脚の改良型。\n安定性を重視し、再設計された。衝撃が\n少なく、軸がぶれにくいのが特徴。",

> 		"tr_explain": "High-tech, improved Jet Boots. Re-designed to improve \nstability. Characterized by slightly shifted center \nand few crashes."

Reviewer: This is a fair bit too long, I'd drop the second sentence entirely as it seems like the third basically reiterates it in more detail. Would also like to see 'improved' moved to the front of the first sentence.
```suggestion
		"tr_explain": "Improved high-tech Jet Boots. Their\nslightly shifted center of gravity is\ndesigned to minimize crashes."
```

	},
	{
		"assign": "116019",
		"jp_text": "ディオレザスイフト",
		"tr_text": "Dio Lezaswift",
		"jp_explain": "新素材で開発された魔装脚の最終形。\n構造を変えることで、さらに軽量化。\n使用時の負担を大幅に軽減させた。",

> 		"tr_explain": "Latest high-tech Jet Boots. \nTheir structure was improved again to make them \neven lighter, drastically reducing load."

Reviewer: Too long.
```suggestion
		"tr_explain": "The latest model of high-tech Jet\nBoots. Their structure was improved\nagain to make them even lighter."
```

	},
	{
		"assign": "116020",
		"jp_text": "レザスイフト",
		"tr_text": "Lezaswift",
		"jp_explain": "新素材を使用し開発された魔装脚。\n密度の高いフォトン刃が二枚付いており\n基礎モデルと比べ、軽量化されている。",

> 		"tr_explain": "High-tech Jet Boots. Made much lighter by addition \nof two high-density Photon blades."

Reviewer: "by **the** addition of", and line breaks should be rearranged a bit, but otherwise this is good.
```suggestion
		"tr_explain": "High-tech Jet Boots. Made much\nlighter by the addition of two high-\ndensity Photon blades."
```

	},
	{
		"assign": "116021",
		"jp_text": "ラウンジブル",
		"tr_text": "Roundibul",
		"jp_explain": "動きを重視した結果、特殊な形状に\nなった魔装脚。重量による負担も\n大きいが、もたらす法撃力は高い。",

> 		"tr_explain": "Jet Boots with unique shape that ephasizes movement. \nTheir propulsion was increased to make up for increased weight."

Reviewer: Too long, also spelling on 'emphasizes'.
```suggestion
		"tr_explain": "Jet Boots that emphasize movement.\nTheir propulsion was increased to\nmake up for their increased weight."
```

	},
	{
		"assign": "116022",
		"jp_text": "ディオラウンジブル",
		"tr_text": "Dio Roundibul",
		"jp_explain": "動き重視のため特殊な形状になった\n魔装脚の最終形。新素材により\n出力が増し、性能が向上した。",

> 		"tr_explain": "Latest model of Jet Boots that emphasize movement. \nTheir output and performance was increased with \nhigh tech materials."

Reviewer: Too long.
```suggestion
		"tr_explain": "Latest model of Jet Boots that\nemphasize movement. Their output\nwas boosted with high-tech materials."
```

	},
	{
		"assign": "116023",
		"jp_text": "ラムダラウンジブル",
		"tr_text": "Lambda Roundibul",
		"jp_explain": "動き重視のため特殊な形状になった\n魔装脚の改良型。内部機構を改良し\n法撃力を高めることに成功した。",

> 		"tr_explain": "Improved Jet Boots that emphasize movement. \nTheir internal mechanism was improved to increase propulsion."

Reviewer: Line breaks need rearranging but otherwise perfect!
```suggestion
		"tr_explain": "Improved Jet Boots that emphasize\nmovement. Their internal mechanism\nwas improved to increase propulsion."
```
